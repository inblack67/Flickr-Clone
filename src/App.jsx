import React, { useEffect } from 'react';
import FlickerState from './context/flicker/FlickerState';

import './App.css';
import 'materialize-css/dist/css/materialize.min.css';
import M from 'materialize-css/dist/js/materialize';

function App() {

  useEffect(() => {
    M.AutoInit();
  },[])

  return (
    <FlickerState>
      <div className="App">
        <h1>hello</h1>
      </div>
    </FlickerState>
  );
}

export default App;
