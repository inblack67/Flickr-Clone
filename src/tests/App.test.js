import React from 'react';
import { render } from '@testing-library/react';
import App from '../App';

describe('App.js', () => {
  it('renders App.js', () => {
    const { asFragment } = render(<App />)
    expect(asFragment).toMatchSnapshot();
  })
})
