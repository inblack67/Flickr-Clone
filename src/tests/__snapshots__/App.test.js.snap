// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App.js renders App.js 1`] = `[Function]`;
